packages/analytics/src/utils/constants.ts

```diff
@@ -17,7 +17,10 @@ export const CONSENT_KEYS = [
   DefaultConsentKeys.MARKETING,
   DefaultConsentKeys.PREFERENCES,
 ];
+
 export const CONSENT_CATEGORIES_PROPERTY = 'consentCategories';
 export const LOAD_INTEGRATION_TRACK_TYPE = 'loadIntegration';
 export const ON_SET_USER_TRACK_TYPE = 'onSetUser';
 export const ANALYTICS_UNIQUE_EVENT_ID = '__uniqueEventId';
```

Comment:
I think this property has the same purpose as the new one just created. Can you check, please? If they are supposed to do the same thing, then delete this one.